docs/templates/translation-issue-template.md

```diff
@@ -1,8 +1,10 @@
 Repository: https://github/[yourname]/open-source-guide
+Preview: https://[yourname].github.io/open-source-guide/
 Locale: `xx-YY`
 
 On the `gh-pages` branch of your fork:
 
+- [ ] Merge the `i18n` branch into `gh-pages` and push the branch to your fork. This should build the website and it should be visible at the preview url above.
```

Comment:
Hey @webknjaz, sorry for the issue. It's been fixed in #348. If you merge the latest changes from the `i18n` branch into your gh-pages branch, it should work now.